### PR TITLE
Navbar & Footer

### PR DESCRIPTION
**Pull request by weepotty**
This PR creates both navbar and footer partials.
I've also moved the vertical white lines/borders styling onto the `<main>` element as this should work when body is a flexbox and the main element flex-grows.

### CODE REVIEW

**Comment by ayushn21** [comment]
I'll come back to this after you merge the previous two PRs and rebase this. There's some outdated stuff in here so it's confusing to review haha. Drop me a line you've brought this up to date :)

File: _includes/world/header.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<header>
+  <nav class="nav">
+    <a href="/world" class="nav__logo">RAILS<span class="nav__logo--outline">WORLD 2023</span></a>
+    <ul>
+      <li><a href="/world/speakers" class="nav__link">Speakers</a></li>
+      <li><a href="#" class="nav__link">Agenda</a></li>
+      <li><a href="/world/faq" class="nav__link">FAQs</a></li>
+      <li><button class="button button--dark button--small"><a href="#">Tickets</a></button></li>
```
Comment (ayushn21) [inline _includes/world/header.html:8]: There's another link within a button here :)

File: _sass/world/modules/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  text-align: center;
+  padding: 0 var(--space-xx-large);
+  border-top: var(--border-white);
+
+  p {
+    font-size: var(--type-x-small);
+    padding-bottom: var(--space-small)
+  }
+
+  bordered-container{
```
Comment (ayushn21) [inline _sass/world/modules/_footer.scss:11]: There should be a space between the CSS selector and the `{`.

**Review by ayushn21 — APPROVED** [review]
A couple of minor things to address but all good otherwise!

**Comment by ayushn21** [comment]
Just realised that the announcement banner is in the main layout so it'll now be shown on every page. I think we only want it on the homepage. Could you please hide it on the other pages using an `if` the same way you did for the top banner on the main site?

Add it to this PR or create a new one as your wish, I'm not too fussed.

**Comment by weepotty** [comment]
> Just realised that the announcement banner is in the main layout so it'll now be shown on every page. I think we only want it on the homepage. Could you please hide it on the other pages using an `if` the same way you did for the top banner on the main site?
> 
> Add it to this PR or create a new one as your wish, I'm not too fussed.

Cool, in the figma it's on every page, i'll double check w Amanda and then make a separate PR

**Pull request merged**